### ISSUE TITLE
magefile.org blocked by Cisco/opendns

### ISSUE THREAD

**Issue by StianOvrevage** [magefile/mage#129]
![image](https://user-images.githubusercontent.com/6630005/44021113-3dba05c2-9ee4-11e8-994d-a7cfa104c53d.png)

**Comment by natefinch** [comment]
This seems like some IT guy got over zealous in his malware settings.  Magefile is a static website hosted by netlify. These kind of enterprise "malware detection" services are notorious for false positives.  I checked my site with google's tools as well as several others, and they all say it's fine.

![image](https://user-images.githubusercontent.com/3185864/44029419-45543db6-9ecb-11e8-9609-6dac86a66f74.png)

Since Umbrella is an enterprise service, maybe you can talk to your IT people about why it's blocking magefile.org?  This isn't something anyone outside your organization would see.

**Comment by StianOvrevage** [comment]
Thanks for checking. 

Yep, it's not a surprise that it's some enterprise security thing. I thought it could be nice to have it here for reference either way. Since it's a Cisco Umbrella thing it probably affects others as well.
